bcda/cli.go

```diff
@@ -211,3 +211,29 @@ func sortCCLFFiles(cclf0, cclf8, cclf9 *[]cclfFileMetadata, skipped *int) filepa
 		return nil
 	}
 }
+
+func deleteDirectory(dirToDelete string) (filesDeleted int, err error) {
+	log.Info(fmt.Sprintf("preparing to delete directory '%v'", dirToDelete))
+	f, err := os.Open(dirToDelete)
```

Comment:
the linter doesn't like this.  where does the dirToDelete var come from?

Reply:
I will nosec it.  It comes from the name of an environment variable that gets passed in as the argument to the command.

Reply:
ok cool.  i guess the other option would be to pull from the environment in this func.  but i haven't looked at it enough to know if that makes sense for the use case.  i'll review again when after whatever change you make!

Reply:
I just added a comment that tells the linter I am aware of this flag and have decided it is ok.

Reply:
`filepath.Clean(dirToDelete)` should prevent the linter warning on it without needing `#nosec`

Reply:
Added.  Thanks. I was not aware of that option.